actioncable/lib/action_cable.rb

```diff
@@ -37,6 +37,10 @@ module ActionCable
     },
     default_mount_path: '/cable'.freeze
   }
+  
+  # This protocol ought to be versioned separately
+  # from the rest of the Gem
+  PROTOCOLS = ["actioncable-protocol-1.0.0".freeze].freeze
```

Comment:
Do we only want to version the ACa transport version, or also provide for an application-level value? I'm thinking of something equivalent to `config.assets.version`...